src/main/java/com/ebay/myriad/executor/MyriadExecutor.java

```diff
@@ -113,6 +114,15 @@ public void disconnected(ExecutorDriver driver) {
 
     @Override
     public void launchTask(final ExecutorDriver driver, final TaskInfo task) {
+        // "task id beginning with "yarn" is a mesos task for yarn container
+        if (task.getTaskId().getValue().startsWith("yarn")) {
```

Comment:
This was why I was afraid things may not work with docker/cgroups isolation, the mailing list seems to indicate this is ok though and will resize the container, https://mail-archives.apache.org/mod_mbox/mesos-user/201412.mbox/%3CCAMKaaZA_EPihHwJT-G6iuCEDZobj_u4QDtfUcucocZmfH=6SMQ@mail.gmail.com%3E.